tests/topotests/conftest.py

```diff
@@ -297,6 +297,59 @@ def check_for_memleaks():
         pytest.fail("memleaks found for daemons: " + " ".join(daemons))
 
 
+def check_for_core_dumps():
+    dumps = []
+    tgen = get_topogen()  # pylint: disable=redefined-outer-name
+    latest = []
+    existing = []
+
+    if tgen is not None:
+        logdir = tgen.logdir
+        if hasattr(tgen, "cores_existing_files"):
+            existing = tgen.cores_existing_files
+        cores = glob.glob(os.path.join(logdir, "*/*.dmp"))
+
+    if tgen is not None:
```

Comment:
just delete this `if` leave the other statement at same indention to be a part of the first `if` :)

Reply:
fixed